README.md

```diff
@@ -64,6 +64,7 @@ module "ocean-aws-k8s-vng" {
   instance_types_filters_min_vcpu                 =   2
   instance_types_filters_root_device_types        =   ["ebs"]
   instance_types_filters_virtualization_types     =   ["hvm"]
+  images                      = {image_id = "ami-12345"}
```

Comment:
please change the alignment and keep 2 values as we have in main.tf

Reply:
Updated 2 values and aligned.